client/my-sites/plans-features-main/components/plan-upsell-modal/index.tsx

```diff
@@ -108,7 +108,7 @@ export default function ModalContainer( props: ModalContainerProps ) {
 	return (
 		<Dialog
 			isBackdropVisible={ true }
-			isVisible={ true }
+			isVisible={ isModalOpen && !! modalType }
```

Comment:
I'm guessing we'd need to remove the lines above, right?

```
	/** Explore using the isVisible flag on Dialog instead. */
	if ( ! isModalOpen || ! modalType ) {
		return;
	}
```

Reply:
Ah yes seems I forgot to push a local commit :D